src/Mages.Core/Runtime/Stringify.cs

```diff
@@ -1,192 +1,185 @@
-﻿namespace Mages.Core.Runtime
```

Comment:
Again this is a full changeset and not just the desired partial diff. Check: Encoding, file endings, and other global settings that essentially swap the full file.